Corona-Warn-App/build.gradle

```diff
@@ -441,6 +441,4 @@ dependencies {
 
     // HCert
     implementation("com.upokecenter:cbor:4.4.1")
-
-    implementation("bouncycastle:bcprov-jdk16:136")
```

Comment:
Why was this previously added and now removed?

Reply:
Works just fine without